atoma-state/src/migrations/20241202121152_node_public_key_rotation.sql

```diff
@@ -2,6 +2,7 @@
 CREATE TABLE IF NOT EXISTS node_public_key_rotations (
     node_small_id            BIGINT  PRIMARY KEY,
     public_key_bytes         BYTEA    UNIQUE NOT NULL,
-    tdx_quote_bytes          BYTEA    NOT NULL,
-    epoch          BIGINT  NOT NULL
+    tee_quote_bytes          BYTEA    NOT NULL,
+    tee_provider             BYTEA    NOT NULL,
```

Comment:
Usually it is a bad practice to change migrations files in place, instead I suggest adding a new migration with a `ALTER TABLE node_public_key_rotations` command